src/services/getUserByEmail.ts

```diff
@@ -0,0 +1,18 @@
+import { User } from '@prisma/client';
+import Axios from './axios';
+
+export const getUserByEmail = async (data: {
```

Comment:
tbh I'm not sure what the difference is supposed to be between a service and a util, but we should agree on it - reply with what would you define them as and then we can discuss?

Reply:
I think the difference between util and service is util is used to fetch the API when util is just doing things that don't have to be rendered